ansible_base/authenticator_plugins/local.py

```diff
@@ -3,22 +3,30 @@
 from django.contrib.auth.backends import ModelBackend
 from django.core.exceptions import ValidationError
 
-from ansible_base.authenticator_plugins.base import AbstractAuthenticatorPlugin
+from ansible_base.authenticator_plugins.base import AbstractAuthenticatorPlugin, BaseAuthenticatorConfiguration
 from ansible_base.models import AuthenticatorUser
 
 logger = logging.getLogger('ansible_base.authentication.local')
 
-# TODO: Figure out how to move this plugin into ansible_base itself
-#       Change the validator to not allow it to be deleted or a second one added
+# TODO: Change the validator to not allow it to be deleted or a second one added
+
+
+class LocalConfiguration(BaseAuthenticatorConfiguration):
+    documentation_url = "https://docs.djangoproject.com/en/4.2/ref/contrib/auth/#django.contrib.auth.backends.ModelBackend"
+
+    def validate_configuration(self, data: dict, instance: object) -> None:
+        if data != {}:
+            raise ValidationError({"configuration": "Can only be {} for local authenticators"})
```

Comment:
So, `validate_configuration` is only on the plugin superclass, not the configuration superclass.

So I think this either needs to be:

```python
    def validate(self, data: dict) -> None:
        if data != {}:
            raise ValidationError({"configuration": "Can only be {} for local authenticators"})
        return data
```

or moved back to the plugin, though just moving it back to the plugin seems to cause weird SQL constraint issues.

Reply:
100% refactored.